### PR TITLE
Add echo

### PR DESCRIPTION
**Pull request by oph-design**
Adds conroller for buildins and the first buildin (echo).
Get well soon.

### CODE REVIEW

File: src/buildins/buildin_controller.c [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+static char	*lower_str(char *str)
+{
+	size_t	i;
+	char	*res;
+
+	i = 0;
+	res = ft_calloc(ft_strlen(str) + 1, sizeof(char));
+	if (res == NULL)
+		return (NULL);
+	while (str[i])
```
Comment (philippschwarzmueller) [inline src/buildins/buildin_controller.c:39]: Don't we have ft_striteri for that case

**Review by philippschwarzmueller — COMMENTED** [review]
If you think it is useful use striteri other than that lgtm

**Comment by philippschwarzmueller** [comment]
Might as well be wrong on the striteri idea, let me know your opinion

**Comment by oph-design** [comment]
I didn't use striteri because it would change the original command and I don't want that (for correct error messages etc.). Lower_str gives you a copy of the command with lower case characters.

**Pull request merged**